pkg/shared/clients/nats/nats_client.go

```diff
@@ -52,7 +52,7 @@ func NewNATSClient(ctx context.Context, natsOptions ...nats.Option) (*NATSClient
 		nats.MaxReconnects(-1),
 		// every one second we will try to ping the server, if we don't get a pong back
 		// after two attempts, we will consider the connection lost and try to reconnect
-		nats.PingInterval(1 * time.Second),
+		nats.PingInterval(20 * time.Second),
```

Comment:
Is 20s too long?

Reply:
We didn't change this in the PR to main.

Reply:
reduced the ping interval to 3 seconds..